src/game/engine/gameptr.cpp

```diff
@@ -46,32 +46,32 @@
 
 #ifndef GAME_DLL
 // Template specialisation to initialise the heaps required for GamePtr.
-template<> FixedIHeapClass *const GamePtr<AircraftTypeClass>::g_Heap = &g_AircraftTypes;
```

Comment:
This change is incorrect, the heaps themselves are not expected the be constant, but the pointers to them are expected to not change after this assignment.